etc/afpd/auth.c

```diff
@@ -827,7 +827,7 @@ int afp_logout(AFPObj *obj, char *ibuf _U_, size_t ibuflen  _U_, char *rbuf  _U_
  */
 int afp_changepw(AFPObj *obj, char *ibuf, size_t ibuflen, char *rbuf, size_t *rbuflen)
 {
-    char username[MACFILELEN + 1], *start = ibuf;
+    char username[MAXUSERLEN + 1], *start = ibuf;
```

Comment:
I don’t think you need the + 1 here since MAXUSERLEN is defined as 256 already.

Reply:
Why did I see it as 255? I modified it

Reply:
Why did I see it as 255? I modified it.
@rdmark